src/biome.rs

```diff
@@ -115,6 +115,20 @@ impl BiomeExtension {
 
     None
   }
+
+  fn require_config_file(&self, settings: &Value) -> bool {
+    settings
+      .get("require_config_file")
+      .and_then(|value| value.as_bool())
+      .unwrap_or(false)
+  }
+
+  fn is_biome_v2(&self) -> bool {
+    zed::npm_package_installed_version(PACKAGE_NAME)
+      .ok()
+      .flatten()
+      .is_none_or(|version| version.starts_with("2."))
```

Comment:
I suggest checking v1 instead, so even future versions like V3, v4, etc. will be compatible.

It's more future proof

Reply:
Yeah it makes sense, fixed in 4054066